frontend/packages/console-shared/src/components/dashboard/inventory-card/InventoryItem.tsx

```diff
@@ -59,10 +59,10 @@ const getStatusGroupIcons = (flags: FlagsObject) => {
 };
 
 const InventoryItem: React.FC<InventoryItemProps> = React.memo(
-  ({ isLoading, title, count, children, error = false, TitleComponent, ExpandedComponent }) => {
+  ({ isLoading, title, titlePlural, count, children, error = false, TitleComponent, ExpandedComponent }) => {
     const [expanded, setExpanded] = React.useState(false);
     const onClick = React.useCallback(() => setExpanded(!expanded), [expanded]);
-    const pluralizedTitle = count !== 1 ? `${title}s` : title;
+    const pluralizedTitle = count === 1 ? title : (titlePlural || `${title}s`);
```

Comment:
We should use the existing `pluralize` utility:

https://github.com/spadgett/console/blob/cc95eed613998a265a48e6bb823c04940edf63da/frontend/public/components/utils/details-page.tsx#L18-L19

Reply:
PR updated - using `pluralize` now